integration-test/src/test/java/org/sagebionetworks/ITDataAccessTest.java

```diff
@@ -137,6 +143,25 @@ public void test() throws SynapseException {
 		
 		managedAR.setIsIDUPublic(true);
 		managedAR = (ManagedACTAccessRequirement) adminSynapse.updateAccessRequirement(managedAR);
+		
+		Instant now = Instant.now();
+		
+		String query = String.format(
+				"select count(*) from accessrequirementsnpashots where"
+						+ " snapshot_date %s and"
+						+ " change_timestamp %s and"
+						+ " instance = '%s' and"
+						+ " id = %s and"
+						+ " change_type = 'UPDATE' and"
+						+ " is_idu_public = true and"
+						+ " concrete_type = '%s'",
+				warehouseHelper.toDateStringBetweenPlusAndMinusFiveSeconds(now),
```

Comment:
This found two rows:
select count(*) from accessrequirementsnapshots where snapshot_date between date('2023-10-13') and date('2023-10-13') and change_timestamp between from_iso8601_timestamp('2023-10-13T21:31:04.622587Z') and from_iso8601_timestamp('2023-10-13T21:31:14.622587Z') and id = 9603006 and change_type = 'UPDATE' and is_idu_public = true and concrete_type = 'org.sagebionetworks.repo.model.ManagedACTAccessRequirement'